infrastructure/server/fam_admin_management_api.tf

```diff
@@ -131,6 +131,9 @@ resource "aws_lambda_function" "fam_admin_management_api_function" {
 
       FC_API_TOKEN = "${var.forest_client_api_api_key}"
       FC_API_BASE_URL = "${var.forest_client_api_base_url}"
+
+      IDIM_PROXY_BASE_URL = "${var.idim_proxy_api_base_url}"
+      IDIM_PROXY_API_KEY = "${var.idim_proxy_api_api_key}"
     }
```

Comment:
New environment variables for IDIM search proxy connection to admin_management lambda.